lib/studies/reply/mail_card_preview.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/layout/adaptive.dart';
+import 'package:gallery/studies/reply/colors.dart';
+import 'package:gallery/studies/reply/mail_view_page.dart';
+
+const _assetsPackage = 'flutter_gallery_assets';
+const _iconAssetLocation = 'reply/icons';
+
+class MailCardPreview extends StatelessWidget {
+  const MailCardPreview({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final isDesktop = isDisplayDesktop(context);
+    final textTheme = Theme.of(context).textTheme;
+
+    return OpenContainer(
+      openBuilder: (context, closedContainer) {
+        return const MailViewPage();
+      },
+      closedShape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(0)),
+      ),
+      closedElevation: 0,
+      closedBuilder: (context, openContainer) {
+        return InkWell(
+          onTap: openContainer,
+          child: Row(
+            children: [
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(20),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Row(
+                        children: [
+                          Text('Google Express', style: textTheme.caption),
+                          Padding(
+                            padding: const EdgeInsetsDirectional.only(
+                              start: 4,
+                              end: 4,
+                            ),
+                            child: Container(
+                              color: Colors.black,
+                              width: 12,
+                              height: 0.5,
+                            ),
+                          ),
+                          Text('15 mins ago', style: textTheme.caption),
+                        ],
+                      ),
+                      const SizedBox(height: 4),
+                      Text('Package Shipped!', style: textTheme.headline6),
+                      const SizedBox(height: 16),
+                      Text(
+                        'Cucumber Mask Facial has shipped',
+                        style: textTheme.bodyText2,
+                      )
+                    ],
+                  ),
+                ),
+              ),
+              Transform.translate(
+                offset: const Offset(-20, -24),
+                child: Row(
+                  children: [
+                    if (isDesktop)
```

Comment:
After the if statement add { and when the if statement finishes add a }. 

https://dart.dev/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures

Reply:
I think dart doesn't let you do this when using the `if` inside of a list like this actually

Reply:
Yeah dart won't let me do this.